src/include/nodes/plannodes.h

```diff
@@ -97,6 +97,8 @@ typedef struct PlannedStmt
 	/* statement location in source string (copied from Query) */
 	ParseLoc	stmt_location;	/* start location, or -1 if unknown */
 	ParseLoc	stmt_len;		/* length in bytes; 0 means "rest of string" */
+
+	int         total_plans;    /* indicates the number of plan nodes in the statement */
```

Comment:
why not unsigned int ?

also, let's name it `plan_nodes_count` or similar as `total_plans` could indicate that there are potentially multiple plans.

Reply:
Makes a lot of sense, will address shortly!

Reply:
src/backend/nodes/gen_node_support.pl doesn't support `unsigned int`:

```
could not handle type "unsigned int" in struct "PlannedStmt" field "plan_nodes_count"
```